src/scripts/convertBreachResolutions.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Executes once
+ * The purpose of the script is to convert all `subscriber.breaches_resolved` to `subscriber.breaches_resolution`
+ * with the goal of deprecating the column
+ */
+
+import Knex from 'knex'
+import knexConfig from '../db/knexfile.js'
+import { getAllBreachesFromDb } from '../utils/hibp.js'
+import { getAllEmailsAndBreaches } from '../utils/breaches.js'
+import { setBreachResolution } from '../db/tables/subscribers.js'
+import { BreachDataTypes } from '../utils/breach-resolution.js'
+const knex = Knex(knexConfig)
+
+const LIMIT = 50 // with millions of records, we have to load a few at a time
+let offset = 0 // looping through all records with offset
+let subscribersArr = []
+
+// load all breaches for ref
+const allBreaches = await getAllBreachesFromDb()
+if (allBreaches && allBreaches.length > 0) console.log('breaches loaded successfully! ', allBreaches.length)
+
+// find all subscribers who resolved any breaches in the past, convert those
+// records into the new v2 format
+do {
+  console.log(`Converting breadches_resolved to breach_resolution - start: ${offset} limit: ${LIMIT}`)
+  subscribersArr = await knex
+    .select('id', 'primary_email', 'breaches_resolved', 'breach_resolution')
+    .from('subscribers')
+    .whereNotNull('breaches_resolved')
+    .limit(LIMIT)
+    .offset(offset)
+
+  console.log(`Loaded # of subscribers: ${subscribersArr.length}`)
+
+  for (const subscriber of subscribersArr) {
+    const { breaches_resolved: v1, breach_resolution: v2 } = subscriber
+    console.debug({ v1 })
+    console.debug({ v2 })
```

Comment:
Testing this locally, `v2` is coming up null for me...

Reply:
v2 can be null, it's expected